mail_tracking/models/res_partner.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    # tracking_emails_count and email_score are non-store fields in order
+    # to improve performance
+    # email_bounced is store=True and index=True field in order to filter
+    # in tree view for processing bounces easier
+    tracking_emails_count = fields.Integer(
+        compute='_compute_tracking_emails_count', readonly=True)
+    email_bounced = fields.Boolean(index=True)
```

Comment:
The name of the misxin mail.blacklist.mixin and the description of this class suggests that it's related to blacklist, not with 'mail tracking' and I don't know if this could confuse someone when using it, maybe another mixing could be created, maybe with the name mail.traking.mixin to give classes that inherit from it aspects related to mail tracking, but do you think other classes need these fields? I mean, do you think other classes need to inherit this new mixing in case we do it?
What do you think?

Reply:
I don't think it's need, as `res.partner` is the only source of recipients.

Reply:
Well, there's replicated functionality for the mailing contacts in mail_tracking_mass_mailing and it could be even used in other models like leads, for instance.

Reply:
I don't have a strong opinion on it anyway :smile:

Reply:
> Well, there's replicated functionality for the mailing contacts

Are you sure? Bounced check is only on partners, not on mass mailing contacts AFAIK. We don't replicate things for mass mailing, but link mail.tracking records to mails sent by mass mailing. If you think that something can be joined under a mixin, then go ahead, as I don't know exact details.

Reply:
res.partner: https://github.com/OCA/social/blob/11.0/mail_tracking/models/res_partner.py#L16
mail.mass_mailing.contact: https://github.com/OCA/social/blob/11.0/mail_tracking_mass_mailing/models/mail_mass_mailing_contact.py#L11

Reply:
Oh, I didn't know! Then it makes sense of course.

Reply:
Yep, good idea!